server/handlers.js

```diff
@@ -112,18 +121,12 @@ instrumentClientJs = function (params, req, res, next) {
   }
 };
 
-
+let noop = function () {};
 export default Handlers = {
```

Comment:
sadness... You optimized some legacy code. Handlers is not exposed anymore if `Conf.IS_ACTIVE === false`, so it does not require this mechanism